src/utils/kafkaFhir.ts

```diff
@@ -76,14 +76,15 @@ export const sendToFhirAndKafka = async (
     // Restore full patient resources to the bundle for sending to Kafka
     Object.keys(newPatientRef).forEach((fullUrl) => {
       const patientData = newPatientRef[fullUrl];
+      const url = `Patient/${patientData.mpiResponsePatient?.id}`;
 
       if (patientData.restoredPatient && bundle.entry) {
         const patientEntry: BundleEntry = {
           fullUrl: fullUrl,
```

Comment:
maybe change this to just "fullUrl," instead of "fullUrl: fullUrl," since the key and the variable have the same name

Reply:
addressed